addon/styles/components/freestyle-subsection.scss

```diff
@@ -1,4 +1,7 @@
 .FreestyleSubsection {
+  &.disabled {
+    display:none;
+  }
```

Comment:
I'm surprised the linter didn't complain that there is no blank line after this `}`. Heh, oh well. No biggie.